app/controllers/users/two_factor_authentication_setup_controller.rb

```diff
@@ -15,6 +15,14 @@ def show
   end
 
   def update
+    if params[:commit] == "Remind me next time"
+      # Ensures the user doesn't go through 2FA check again.
+      request.env["warden"].session(:user)[TwoFactorAuthentication::NEED_AUTHENTICATION] = false
```

Comment:
could you factor this and line 31 into one method instead of having the same 2 lines twice?